### PR TITLE
Protect: Changelog and Readme updates for 1.0.2 release

### PR DESCRIPTION
**Pull request by miguelxpn**
#### Changes proposed in this Pull Request:

* This is creating the changelog for the new 1.0.2 release for Jetpack Protect.

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Nothing to test really.

### CODE REVIEW

File: projects/js-packages/babel-plugin-replace-textdomain/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.12] - 2022-07-22
+
```
Comment (coder-karen) [inline projects/js-packages/babel-plugin-replace-textdomain/CHANGELOG.md]: A quick suggestion here is to make sure there aren't any blank changelog entries - in cases like this it's generally just package dependency updates so you can copy-paste the previous entry:
```suggestion
## [1.0.12] - 2022-07-22
### Changed
- Updated package dependencies.
```
There are a number of other packages with blank entries too.

**Review by coder-karen — CHANGES_REQUESTED** [review]

**Review by coder-karen — DISMISSED** [review]

File: projects/plugins/protect/readme.txt [hunk 2]
```diff
@@ -109,19 +109,11 @@ The features of the new Jetpack Protect plugin are not included in the [Jetpack
...
-#### Added
-- Added new two FAQ entries
-- Protect: record even just after the site is registered
-
+### 1.0.2 - 2022-07-22
 #### Changed
-- Moved normalization of the Protect status report data to the server side.
-- Renamed `master` references to `trunk`
-- Reorder JS imports for `import/order` eslint rule.
+- Updated package dependencies.
```
Comment (jeherve) [inline projects/plugins/protect/readme.txt]: This should probably be removed from the readme.txt file; it's not very useful for end-users to know.

File: projects/plugins/protect/readme.txt [hunk 3]
```diff
@@ -109,19 +109,11 @@ The features of the new Jetpack Protect plugin are not included in the [Jetpack
...
 
 #### Fixed
-- Fixed bug that would not display Core vulnerabilities.
-- Removed legacy code and documentation and added new docs on the debug helper plugin.
-- Fixed recommendation for plugins that don't have a fix yet
-- Protect: Fix visual issue of the Interstitial page
+- Fix protect admin toolbar icon display when Jetpack enabled and connected
```
Comment (jeherve) [inline projects/plugins/protect/readme.txt]: Typically, the readme.txt file is mostly read by end-users, so I would recommend using wording that is as user-friendly and clear as possible there. Something like:

```suggestion
- Admin toolbar: ensure the toolbar icon is properly displayed, even when the Jetpack plugin is enabled and connected.
```

File: projects/plugins/protect/readme.txt [hunk 4]
```diff
@@ -109,19 +109,11 @@ The features of the new Jetpack Protect plugin are not included in the [Jetpack
...
+- Minor bug fix - added isset() checks for report data properties
```
Comment (jeherve) [inline projects/plugins/protect/readme.txt]:
```suggestion
- General: avoid PHP notices when fetching reports about your site.
```

**Review by jeherve — CHANGES_REQUESTED** [review]
We'll most likely need to update the stable tag here as part of this PR:
https://github.com/Automattic/jetpack/blob/protect/1.0.2-changelog-readme/projects/plugins/protect/readme.txt#L7

It can be updated to 1.0.1, the current stable, and then it will be updated to 1.0.2 as part of the release process in a few hours.

**Review by coder-karen — APPROVED** [review]

**Comment by jeherve** [comment]
Closing in favor of #25290. This PR is a bit out of date, it misses the changes introduced in #25279.

**Pull request closed without merging**